docs/sections/getting_started.md

```diff
@@ -1,8 +1,8 @@
 # Getting Started
 
-Popper is a workflow execution engine based on [Github Actions][gha] 
-(GHA) written in Python. With Popper, you can execute [HCL 
-syntax][ghalang] workflows locally on your machine without having 
+Popper is a workflow execution engine based on [Github Actions][gha]
```

Comment:
since these changes (remove blank space from end of line) are only cosmetic ones, they really are not relevant to this PR. Would it be possible to leave them out of this PR?

Reply:
To be honest I don't know why it shows as changes. I actually didn't touch anything here. Any tips on what I should do to stop it from showing I'm making changes?

Reply:
From further investigation, I believe an extension that I have enabled or a setting that automatically removes blank lines.

Reply:
Yes, I faced this similar issue back, These are due to python extensions enabled in your IDE.

Reply:
Oh wow! @vipulchhabra99. So what was your workaround? Because deactivating my extensions isn't gonna be an option, lol.

Reply:
you can leave them out, but in a separate commit, so that we can review the actual changes separately, and then have the cosmetic changes on another